devops/ansible/vars/default.yml

```diff
@@ -0,0 +1,16 @@
+# General app vars
```

Comment:
End roles. Config files are relevant but should be similar to what we used with our Imposter playbook.